rasa/core/tracker_store.py

```diff
@@ -105,7 +118,16 @@ def create(
         import sqlalchemy.exc
 
         try:
-            return AwaitableTrackerStore(_create_from_endpoint_config(obj, domain, event_broker))
+            tracker_store = _create_from_endpoint_config(obj, domain, event_broker)
+            if not check_if_tracker_store_async(tracker_store):
+                rasa.shared.utils.io.raise_warning(f"Tracker store implementation {tracker_store.__class__.__name__} "
```

Comment:
Small one: we have a specific method `rasa.shared.utils.io.raise_deprecation_warning` for deprecation warnings. We can state it is now deprecated and will be removed in 4.0.0.

Reply:
Done!